README.md

```diff
@@ -103,6 +98,12 @@ running before and after benchmarking to inform you of benchmarking progress),
 which can be specified in the `PRE_EXECUTION_CMDS` and `POST_EXECUTION_CMDS`
 settings in your Krun config file.
 
+Commands in each list are run, in order, using the `krun` user's shell (e.g.
+`bin/sh`). If a command fails, Krun stops execution immediately without running
```

Comment:
The krun user gets the default shell for the platform upon which we are running:
https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/krun/platform.py#L521-L523

(The benchmarks themselves are always run with dash though)

Reply:
That's what I said, right?

Reply:
You said `bin/sh`, but that's not necessarily the default shell.

The default shell on Debian is configured by `/etc/default/useradd`.

Perhaps we should specify /bin/sh as the user's shell at creation time?

Reply:
The "e.g." is crucial.